### ISSUE TITLE
Interest in collaboration

### ISSUE THREAD

**Issue by ewels** [SciLifeLab/NGI-CellRaiser#2]
Hi everyone,

Please add a comment to this thread if you're interested in helping out with developing this pipeline.

Consolidated list at top of thread for reference (I'll update this as people reply):

- QBiC Tübingen (contact: @apeltzer, @sven1103, @subwaystation)
- NGI Stockholm (contact: @ewels)
- NGI Uppsala (contact: @jessieno)
- Sanger Institute (contact: @wikiselev)

Perhaps we can have a conference call / google hangout in January to discuss how to proceed.

Phil

**Comment by subwaystation** [comment]
Hi Phil,
thanks for the organization!
From the QBiC side I am also very interested:
- QBiC Tübingen: @subwaystation

**Comment by sven1103** [comment]
Hey Phil, 

conference call in January sounds good, probably in the week of 8th to 12th of January, as some of us are on leave.

Sven

**Comment by wikiselev** [comment]
Thanks, Phil, I am in with all my hands!

Sanger Institute: @wikiselev

**Comment by ewels** [comment]
Brilliant stuff!

I had a few hours free this afternoon so I made a new mini project: https://github.com/SciLifeLab/NGI-NFcookiecutter

It's a templating system for creating new Nextflow pipelines. Could make things a little easier to get started with here.

**Comment by sven1103** [comment]
wtf, it works perfectly, just created a test workflow with cookiecutter! Thanks Phil, we will use it for this repo.

**Comment by glrs** [comment]
Hi Phil,

Even though I am not sure how much I am able to contribute to this, I would be glad to get involved!
ESCG: @glrs

Anastasios

**Comment by ewels** [comment]
ESCG = SciLifeLab Eukaryotic Single Cell Genomics platform (Stockholm):
 https://www.scilifelab.se/facilities/eukaryotic-single-cell-genomics/

**Comment by svennahnsen** [comment]
Has anybody looked at SCANPY (https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1382-0)?

**Comment by wikiselev** [comment]
yes, I've seen SCANPY, looks like another good candidate for a NF pipeline. But probably it's worth waiting a bit until it becomes more widely used.
